modules/pel/peltool/peltool.py

```diff
@@ -365,14 +369,35 @@ def listOption(path: str, config: Config, extension: str, rev: bool =False):
             file_list.append(file)  ## create list of file names
         # Only process top level directory
         break
-    file_list.sort(reverse=rev) 
+    file_list.sort(reverse=rev)
+    return root, file_list
+
+def listOption(path: str, config: Config, extension: str, rev: bool = False):
+    root, file_list = getFileList(path, extension, rev)
     final_summary = {}
     for file in file_list:
         eid, summary = extractAndSummarizePEL(os.path.join(root, file), config)
         if eid :
             final_summary[eid] = summary
     print(prettyPrint(json.dumps(final_summary, indent=4) , desiredSpace = 29))
 
+def extractAllPELsData(path: str, config: Config, extension: str, rev: bool = False):
+    config.serviceable_only = True
```

Comment:
Though it's fairly obvious what this function does, it may be good to add comments at some point so it's consistent with the other functions you've added.